Description: Use /etc/ntpviz for header and footer
 This provides conffiles in a reasonable location for the administrator to
 customize the header and footer for ntpviz.
Forwarded: not-needed
Origin: vendor
Author: Richard Laager <rlaager@wiktel.com>
Last-Update: 2016-12-04
--- a/docs/includes/ntpviz-body.txt
+++ b/docs/includes/ntpviz-body.txt
@@ -165,11 +165,15 @@
 
 The first file is named 'header'.  The contents of that file will be
 added almost at the top of the body on the generated index page.
+As a special exception, the default cron job in the ntpsec-ntpviz package
+has symlinks such that it uses `/etc/ntpviz/header.html`.
 
 This is the place to put links to other web pages, or headline notes.
 
 The second file is named 'footer'.  The contents of that file will be
 added almost at the bottom of the body on the generated index.
+As a special exception, the default cron job in the ntpsec-ntpviz package
+has symlinks such that it uses `/etc/ntpviz/footer.html`.
 
 It is suggested that notes on the server be included in the footer
 file: OS, versions, CPU speed, etc.  You may also put links there.
